es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -0,0 +1,2010 @@
+                    type: long
+                  state:
+                    type: keyword
+                  submitter_id:
+                    type: keyword
+                  therapeutic_agents:
+                    type: keyword
+                  treatment_anatomic_site:
+                    type: keyword
+                  treatment_id:
+                    type: keyword
+                  treatment_intent_type:
+                    type: keyword
+                  treatment_or_therapy:
+                    type: keyword
+                  treatment_outcome:
+                    type: keyword
+                  treatment_type:
+                    type: keyword
+                  updated_datetime:
+                    type: keyword
+              tumor_grade:
+                type: keyword
+              tumor_stage:
+                type: keyword
+              updated_datetime:
+                type: keyword
+              vascular_invasion_present:
+                type: keyword
+              vital_status:
+                type: keyword
+              year_of_diagnosis:
+                type: long
+          disease_type:
+            type: keyword
+          exposures:
+            type: nested
+            properties:
+              alcohol_history:
+                type: keyword
+              alcohol_intensity:
+                type: keyword
+              bmi:
+                type: long
+              cigarettes_per_day:
+                type: long
+              created_datetime:
+                type: keyword
+              exposure_id:
+                type: keyword
+              height:
+                type: long
+              pack_years_smoked:
+                type: long
+              state:
+                type: keyword
+              submitter_id:
+                type: keyword
+              tobacco_smoking_onset_year:
+                type: long
+              tobacco_smoking_quit_year:
+                type: long
+              tobacco_smoking_status:
+                type: keyword
+              updated_datetime:
+                type: keyword
+              weight:
+                type: long
+              years_smoked:
+                type: long
+          family_histories:
+            type: nested
+            properties:
+              created_datetime:
+                type: keyword
+              family_history_id:
+                type: keyword
+              relationship_age_at_diagnosis:
+                type: long
+              relationship_gender:
+                type: keyword
+              relationship_primary_diagnosis:
+                type: keyword
+              relationship_type:
+                type: keyword
+              relative_with_cancer_history:
+                type: keyword
+              state:
+                type: keyword
+              submitter_id:
+                type: keyword
+              updated_datetime:
+                type: keyword
+          files:
```

Comment:
I am surprised to see another `files` node here, it's like: `case.files.cases.files`. We should only have `case.files.cases`. Checked on `case` document, there is not such fourth level of `files`.

Should remove this in the mapping and try load JSON docs into ES in `strict` mode, if loading passed, it's safe to remove this `files` node.